src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplateBuilder.java

```diff
@@ -260,7 +265,20 @@ public Pod build() {
         envVars.putAll(jnlp.getEnv().stream().collect(Collectors.toMap(EnvVar::getName, Function.identity())));
         jnlp.setEnv(new ArrayList<>(envVars.values()));
         if (jnlp.getResources() == null) {
-            jnlp.setResources(new ContainerBuilder().editOrNewResources().addToRequests("cpu", new Quantity(DEFAULT_JNLP_CONTAINER_CPU_REQUEST)).addToRequests("memory", new Quantity(DEFAULT_JNLP_CONTAINER_MEMORY_REQUEST)).endResources().build().getResources());
+
+            containerBuilder = new ContainerBuilder();
```

Comment:
Probably don't need to have a call to editOrNewResources for each request/limit.

```suggestion
            ResourcesNested<ContainerBuilder> containerResources = new ContainerBuilder().editOrNewResources();
```

Reply:
fix